APR-Dragonflight/Dragonflight_Alliance.lua

```diff
@@ -491,7 +495,6 @@ if (APR.Faction == "Alliance") then
                },
                ["Range"] = 5,
                ["ExtraLineText"] = "PRESS_1",
-               ["Dontskipvid"] = 1,
```

Comment:
This needs testing. On beta, when the skip was enabled, this caused the cooldown of the ability that you need to press for the next part of the quest to not reset, meaning you had to wait until it counted all the way down before you could press it. If you don't skip the video, the cooldown resets when you land. In other words, it was faster to let the cutscene play, than to skip it.